nixos/modules/programs/zsh/zsh.nix

```diff
@@ -67,9 +67,7 @@ in
       };
 
       promptInit = mkOption {
-        default = ''
-          autoload -U promptinit && promptinit && prompt walters
```

Comment:
why did you remove the default here? even tho a custom prompt will be set afterwards the `walters` prompt will always get loaded.